src/index.ts

```diff
@@ -61,10 +61,12 @@ function transformCode(this: TransformPluginContext, srcCode: string, id: string
     },
     // Only keep primitive properties
     inputSourceMap: JSON.parse(JSON.stringify(this.getCombinedSourcemap())),
-  });
+  }
+
+  const { code, map } = (await transformAsync(srcCode, babelConfig))!
 
   // Required to cast to correct mapping value
-  return { code, map: JSON.parse(JSON.stringify(map)) };
```

Comment:
Not sure about this change, it was previously added to fix an issue between Babel and Vite's source map handling. Not sure if Vite fixed this or if it could be a problem after running the instrumentation in some cases. It is mentioned before in this issue: https://github.com/iFaxity/vite-plugin-istanbul/issues/1#issuecomment-757575648